packages/vitest/src/types/config.ts

```diff
@@ -184,6 +184,13 @@ export interface InlineConfig {
    */
   api?: boolean | number
```

Comment:
should we change `api` to `port`?

Reply:
breaking change might occur

Reply:
this flag is to register the api, so it is ok...